Assets/Tests/PlayMode/Extensions/ListExtensionsTests.cs

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+namespace PlayEveryWare.EpicOnlineServices.Tests.Extensions
+{
+    using NUnit.Framework;
+    using PlayEveryWare.EpicOnlineServices.Extensions;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    [TestFixture]
+    public class ListExtensionsTests
+    {
+        /// <summary>
+        /// Test to verify that the Shuffle method changes the order of elements in the list.
+        /// </summary>
+        [Test]
+        public void Shuffle_ChangesOrderOfElements()
+        {
+            // Arrange
+            List<int> originalList = Enumerable.Range(1, 100).ToList();
+            List<int> shuffledList = new(originalList);
+
+            // Act
+            shuffledList.Shuffle();
+
+            // Assert
+            Assert.AreNotEqual(originalList, shuffledList,
```

Comment:
not review: I wonder if this works as intended? Does this just compare the pointer to the originalList and the shuffledList?

So I made a tiny test;

```c#
        [Test]
        public void AreEqualTest()
        {
            List<int> originalList = Enumerable.Range(1, 10).ToList();
            List<int> otherList = Enumerable.Range(1, 10).ToList();
            Assert.AreNotEqual(originalList, otherList);
        }
```

This test failed; the two arrays are equal, even though they're different pointers. So there's probably some equality operator being run. So you have it correct here!